src/test/scala/uk/gov/tna/metadata/schema/validator/SharepointDataLoadSchemaSpec.scala

```diff
@@ -0,0 +1,73 @@
+package uk.gov.tna.metadata.schema.validator
+
+import com.networknt.schema.{InputFormat, ValidationMessage}
+import scala.jdk.CollectionConverters.*
+import java.util
+import org.scalatest.matchers.should.Matchers.*
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.io.File
+import java.nio.file.Files
+
+class SharepointDataLoadSchemaSpec extends BaseSpec {
+
+  "sharepoint data load schema validation" should {
+    "pass when all required properties present" in {
+      val schemaPath = "metadata-schema/dataLoadSharePointSchema.schema.json"
+      val dataPath = "/data/sharepointDataLoad.json"
+      val schemaInputStream = Files.newInputStream(new File(schemaPath).toPath)
+      val schema = getJsonSchemaFromStreamContentV7(schemaInputStream)
+      val dataInputStream = getClass.getResourceAsStream(dataPath)
+      val node = getJsonNodeFromStreamContent(dataInputStream)
```

Comment:
Can you move the above common statements to a separate method?

  ```
private def createSchema(dataPath: String) = {
    val schemaPath = "metadata-schema/dataLoadSharePointSchema.schema.json"
    val schemaInputStream = Files.newInputStream(new File(schemaPath).toPath)
    val schema = getJsonSchemaFromStreamContentV7(schemaInputStream)
    val dataInputStream = getClass.getResourceAsStream(dataPath)
    val node = getJsonNodeFromStreamContent(dataInputStream)
    (schema, node)
  }
```
And Call it from you all test methods:

      `val (schema: JsonSchema, node: JsonNode) = createSchema(dataPath)
`

Reply:
Good point.